calico_node/startup/startup.go

```diff
@@ -906,6 +919,69 @@ func ensureDefaultConfig(ctx context.Context, cfg *apiconfig.CalicoAPIConfig, c
 	return nil
 }
 
+// ensureMigrated ensures any data migration needed is done.
+func ensureMigrated(ctx context.Context, cfg *apiconfig.CalicoAPIConfig, c client.Interface) error {
```

Comment:
Any reason this shouldn't be part of the migration package instead?  Is there a reason that the checks it does are specific to startup.go?  Context: I'm trying to get migration into Typha and I don't want to have to copy/paste a bunch of code.

Reply:
Sure I can move it there (or you if you want to copy/move it in with something you're already doing).  I hadn't considered that this would need to be done anywhere else is why it is here.